designs/cloudstack-multiple-endpoints.md

```diff
@@ -163,6 +177,27 @@ api-url    = http://172.16.0.3:8080/client/api
 ...
 ```
 
+1. `Global`, `AvailabilityZone2`, and `AvailabilityZone3` are entirely arbitrary names that are unnecessary.
+2. If these tags mismatch between the name in the AvailabilityZone and here there will be an error.
+3. If we use validation, then we can prevent mismatched names, but what about modifying this secret in an existing cluster?
+    - Validation doesn't operate on an existing cluster unless the secret is updated through EKS-A.
+    - If an existing cluster wants to update a secret, the entire secret must be modified together. If this results in
+    an error, the entire set of AvalailabilityZones are at risk.
+4. To do validation of the secret ini with multiple tags the DataCenter needs to match the tags to the names of its
+   AvailabilityZones. This is fine, but with yaml the secrets can be iterated on and checked individually without
+   concern for the tags.
```

Comment:
we still need a way to map from the secrets in the yaml file to the AvailabilityZones defined in the datacenterconfig to support failing fast in case a user has made a mistake. We can do that via the ACS url or a more friendly name. It looks like you're proposing using the ACS url, is that correct?